pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -332,8 +334,29 @@ func (k *Bootstrapper) WaitForNode(cfg config.ClusterConfig, n config.Node, time
 		glog.Infof("%s is not a control plane, nothing to wait for", n.Name)
 		return nil
 	}
+
+	// starting from minikube v1.10 checking for node pressure is a reuqired check
+	out.T(out.HealthCheck, "Verifying Kubernetes Components:")
+	out.T(out.OptionVerify, "verifying node conditions ...")
```

Comment:
Can we remove this line? It seems noisy, and doesn't inform the user any more than the previous line item.

Reply:
leave this for a follow up PR to refresh the UI